### PR TITLE
ci: trigger Algolia crawl after deploy

### PR DESCRIPTION
**Pull request by SMillerDev**
Closes GH-671

### CODE REVIEW

**Comment by SMillerDev** [comment]
Should set the secrets before merging this

File: .github/workflows/scheduled.yml [hunk 1]
```diff
@@ -202,8 +202,25 @@ jobs:
...
+      - name: Algolia crawler creation and recrawl
+        uses: algolia/algoliasearch-crawler-github-actions@v1
+        id: crawler_push
+        with:
+          override-config: true
+          site-url: ${{ needs.deploy.outputs.deploy_url }}
+          crawler-user-id: ${{ secrets.CRAWLER_USER_ID }}
+          crawler-api-key: ${{ secrets.CRAWLER_API_KEY }}
+          algolia-app-id: ${{ secrets.ALGOLIA_APP_ID }}
+          algolia-api-key: ${{ secrets.ALGOLIA_API_KEY }}
```
Comment (MikeMcQuaid) [inline .github/workflows/scheduled.yml:226]:
```suggestion
          algolia-api-key: ${{ secrets.ALGOLIA_API_KEY }}

```

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by Rylan12** [comment]
@SMillerDev it looks like this is currently failing (apologies if you're already aware and working on it)

**Comment by SMillerDev** [comment]
Yeah, Mike messaged me about it. I'm not sure why though so I'm planning to just silence the errors while I figure it out. Algolia already does a daily scan too.

**Pull request merged**